### DISCUSSION TITLE
v-autocomplete not working with id-text arrays

### DISCUSSION THREAD

**Discussion by xdr-git** [vuetifyjs/vuetify#17416] [Q&A]
Hi, i have a problem,
The v-autocomplete component not working when i use an array with more of one properties, like: [id: 1, name: 'Item Title'].
The v-model its a number (id), and show correct title for this element inside array, for example: v-model="1" (show Item Title n the component) but when i typing, i dont see any result.
I clarify, when I pass a simple array like ['itemtext', 'itemtext2'] if it works! and i use the item-title and item-value correctly.
Help me please!, thanks

**Comment by yuwu9145** [comment]
Can you provide a reproduction link please?

**Reply by xdr-git** [reply]
On the Vuetify website (in their documentation), there is an example shown in the v-autocomplete component, specifically in the filter example (https://vuetifyjs.com/en/components/autocompletes/#filter). 
When you type "Florida," the component doesn't show any results, even though Florida is clearly in the list. Thank you for your interest!

**Reply by KaelWD** [reply]
#16789
